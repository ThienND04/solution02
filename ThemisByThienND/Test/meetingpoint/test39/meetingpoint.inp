8 10 4
3 4 2
8 2 19
3 6 1
6 4 1
2 1 1
8 7 13
8 1 20
3 7 3
5 8 14
7 3 3
2 2
8 9
7 1
5 6
