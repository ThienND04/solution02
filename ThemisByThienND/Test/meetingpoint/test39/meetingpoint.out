38
160
19
100
